pyproject.toml

```diff
@@ -43,11 +43,11 @@ include = [
   "unittests",
 ]
 [tool.pdm.dev-dependencies]
-doc = ["pelican", "docutils >=0.16", "markupsafe >= 2.0.0" ]
+doc = ["pelican", "docutils >=0.16", "markupsafe >= 2.0.0", "meson" ]
 #doc = ["pelican", "doxygen"]
 
 [build-system]
-requires = ["setuptools", "pdm-backend", "meson @ git+https://github.com/robtaylor/meson@mac-homebrew",  "ninja", "meson-python == 0.13.2"]
+requires = ["setuptools", "pdm-backend", "meson @ git+https://github.com/robtaylor/meson@mac-homebrew",  "ninja", "meson-python == 0.13.2", "meson"]
```

Comment:
The meson dependency is already there (temporarily pointing at my branch until new release of meson)
Was there a reason you needed to add these?

Reply:
No... Those are remnants of failed attempts to get the right meson installed. Will remove them.